### PR TITLE
modularize the function and add TODOs

### PR DESCRIPTION
**Pull request by Ohyeon5**
modularized the steps, and defined TODOs to make this app functional :)

### CODE REVIEW

File: ui/app.py [hunk 1]
```diff
@@ -58,22 +62,29 @@ def tldr(text_input):
...
         st.image(DEFAULT_IMG_URL, width=500)
 
 
 if __name__ == "__main__":
     init()
-    text_input = input_initial_data()
-    text = tldr(text_input)
-    dalle2(text)
+    question = get_search_question()
+    # TODO: semantic search using question and get list of concerns
```
Comment (Ohyeon5) [inline ui/app.py:81]: There are 2 main parts where we need the openAI APIs implemented :) Shall we split the job?

File: ui/app.py [hunk 2]
```diff
@@ -58,22 +62,29 @@ def tldr(text_input):
...
+    results = ["batteries", "aluminium usage", "electricity demand", "..."]
+    return_search_results(results)
+    # TODO: search results to suggested action
```
Comment (Ohyeon5) [inline ui/app.py:84]: and this one!

File: ui/app.py (hunk 1 above)
Comment (metinaydar) [reply ui/app.py:81]: I'm taking the part:     # TODO: semantic search using question and get list of concerns

File: ui/app.py (hunk 2 above)
Comment (metinaydar) [reply ui/app.py:84]: maybe for @lfhvn ? :) 
searching policies for suggesting some actions via https://github.com/climatepolicyradar/open-data

**Review by metinaydar — APPROVED** [review]

**Pull request merged**